Projects/Foundations/Schemas/Docs-Nodes/L/LAN/LAN-Network/lan-network.json

```diff
@@ -120,24 +150,34 @@
                     "language": "DE",
                     "text": "Verwaltung der Speicherung der Daten, die von den Bots erzeugt werden, die Sie als Outputs ausführen. Dazu gehört auch die Verwaltung der physischen Speicherorte, an denen die von den Bots erzeugten Datenprodukte gespeichert werden.",
                     "updated": 1639570975495
+                },
+                {
+                    "language": "EL",
+                    "text": "Για να διαχειριστείτε την αποθήκευση των δεδομένων που παράγονται από τα ρομπότ που εκτελείτε ως αποτελέσματα. Αυτό περιλαμβάνει τη διαχείριση των φυσικών τοποθεσιών στις οποίες βρίσκονται τα προϊόντα δεδομένων που παράγονται από ρομπότ.",
+                    "updated": 1641827779980
                 }
             ]
         },
         {
             "style": "Warning",
             "text": "At this stage, the system does not implement any form of security measures, therefore, Superalgos is to be run in the context of a restricted Local Area Network only, unless you implement your own network security.",
             "translations": [
-              {
+                {
                     "language": "RU",
                     "text": "На данном этапе в системе не реализованы какие-либо меры безопасности, поэтому Superalgos следует запускать только в контексте ограниченной локальной сети, если вы не реализуете собственную сетевую безопасность.",
                     "updated": 1638337639123
-              },
+                },
                 {
                     "language": "DE",
                     "text": "In diesem Stadium implementiert das System keinerlei Sicherheitsmaßnahmen, daher sollte Superalgos nur im Rahmen eines eingeschränkten lokalen Netzwerks betrieben werden, es sei denn, Sie implementieren Ihre eigene Netzwerksicherheit.",
                     "updated": 1639570995473
+                },
+                {
+                    "language": "EL",
+                    "text": "Σε αυτό το στάδιο, το σύστημα δεν εφαρμόζει κανενός είδους μέτρα ασφαλείας, επομένως, το Superalgos πρέπει να εκτελείται μόνο στο πλαίσιο ενός περιορισμένου τοπικού δικτύου, εκτός εάν εφαρμόζετε τη δική σας ασφάλεια δικτύου.",
+                    "updated": 1641827797173
                 }
             ]
         }
     ]
-}
+}
```

Comment:
Here it seems like the Brackets got moved. I don`t know if it has an impact. @harrellbm what do you think?

Reply:
@gitforkgit as long as the same brackets are still there the indentation of them doesn't really matter